packages/docs/pages/usage/search/sorting.mdx

```diff
@@ -0,0 +1,79 @@
+# Sorting
+
+To sort, Orama uses the properties defined in the `schema` to know on which properties you want to sort.
+
+```javascript copy
+const db = await create({
+  schema: {
+    title: 'string',
+    year: 'number',
+    isTop: 'boolean',
+    meta: {
+      tag: 'string',
+      rating: 'number',
+      favorite: 'boolean',
+    },
+  }
+})
+const results = await search(db, {
+  term: 'prestige',
+  sort: {
+    property: 'title', // or 'year', 'isTop'
+  }
+})
+```
+
+Orama supports sorting on 'string' and 'number'.
```

Comment:
Non blocking: in the future we can also support boolean pretty easily I guess once we define the order.

Reply:
Why should I sort `true` and `false` values?

Reply:
For exactly this reason. Lol. You might want to have the docs with some field set to `true` and then `false` true.
Think about having the or `inPromotion` property. In the results, regardless of the score, you might want to have the promoted items first.